.devcontainer/devcontainer.json

```diff
@@ -6,6 +6,8 @@
 	"image": "mcr.microsoft.com/devcontainers/python:1-3.11-bullseye",
 	"features": {
 		"ghcr.io/devcontainers/features/azure-cli:1": {},
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {},
```

Comment:
In a discussion I had with @rngadam, he stated that he prefers not to add `act` on backend repositories and instead leave that to the devops repositories.

Reply:
Wich one do I need to not have? All of the features? Or just a specific one?

Reply:
Done